docs/assets/software-catalog/.$entity.svg.bkp

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this a backup file from your editor? If so, feel free to add `*.bkp` to `.gitignore` as part of this PR and remove the file.

Reply:
Ups yeah sorry. I removed it.